app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -85,46 +69,56 @@ To be eligible for a scholarship, you’ll need at least one of the following:
+|-----------------------------------------------------------------------| -------: | ---------------: |
+| **Art and design**                                                    | $bursaries_postgraduate_artanddesign$  |                  |
+| **Biology**                                                           | $bursaries_postgraduate_biology$  |                  |
+| **Chemistry**                                                         | $bursaries_postgraduate_chemistry$  | $scholarships_chemistry$          |
+| **Computing**                                                         | $bursaries_postgraduate_computing$  | $scholarships_computing$          |
+| **Design and technology**                                             | $bursaries_postgraduate_designandtechnology$  |                  |
+| **English**                                                           | $bursaries_postgraduate_english$  |                  |
+| **Geography**                                                         | $bursaries_postgraduate_geography$  |                  |
+| **Languages**<br/> (French, German and Spanish only)                  | $bursaries_postgraduate_languagesfrenchgermanspanish$  | $scholarships_languagesfrenchgermanspanish$      |
+| **Languages**<br/> (all other languages, including ancient languages) | $bursaries_postgraduate_languagesother$  |                  |
+| **Maths**                                                             | $bursaries_postgraduate_maths$  | $scholarships_maths$         |
+| **Music**                                                             | $bursaries_postgraduate_music$  |                  |
+| **Physics**                                                           | $bursaries_postgraduate_physics$  | $scholarships_physics$        |
+| **Religious education**                                               | $bursaries_postgraduate_religiouseducation$  |                  |
 
-The amount you receive depends on the year that you start your QTS course. You do not get a bursary in every year of an undergraduate course. Depending on the type of course you do, you may get the bursary in your final year or final two years.
+Table caption: Available postgraduate bursaries and scholarships
 
-### Undergraduate bursary if you're former military personnel
+## If you think you're not eligible
 
-You may be eligible for a $bursaries_veteran_total$ tax-free undergraduate bursary for veterans if you’re:
+If you're not eligible for a bursary or scholarship, there are other funding options. 
 
-* training to teach secondary biology, chemistry, computing, languages, maths, or physics
-* doing an undergraduate degree leading to QTS in England
+You can:
 
-[Find out more about how to get support training to teach if you're a veteran](/funding-and-support/if-youre-a-veteran).
+* find out more about [tuition fees and maintenance loans](/funding-and-support/tuition-fees)
+* talk to a [teacher training adviser](/help-and-support)
```

Comment:
I think these two bullets are not really equivalent - it sounds like TTAs are a funding option?

I think the wording of the TTA bullet needs changing to make it clear they can give **advice** and possibly that the bullets need removing altogether and just replaced by normal sentences?